streams/writable-streams/aborting.js

```diff
@@ -392,12 +424,13 @@ promise_test(t => {
     }
   });
   const writer = ws.getWriter();
+
   return writer.ready.then(() => {
     return Promise.all([
-      promise_rejects(t, error1, writer.write('a'), 'writer.write() should reject with error from underlying write()'),
-      promise_rejects(t, new TypeError(), writer.close(),
-                      'writer.close() should reject with error from underlying write()'),
-      writer.abort()
+    promise_rejects(t, error1, writer.write('a'), 'writer.write() should reject with error from underlying write()'),
+    promise_rejects(t, new TypeError(), writer.close(), 'writer.close() should reject with error from underlying write()'),
+    // Write() rejecting on our UnderlyingSink should error the stream, so abort should be rejected when we try to call it.
+    promise_rejects(t, error1, writer.abort(), 'writer.abort() should reject with error from underlying write()')
```

Comment:
I don't believe this is a correct change per spec. Perhaps tracing through the reference implementation would help us understand it better.